apps/task/schema.py

```diff
@@ -41,22 +46,87 @@ class Meta:
 
 
 class TimeEntryType(DjangoObjectType):
+    duration = graphene.String()
+    day_total = graphene.String()
+
     class Meta:
         model = TimeEntry
         fields = '__all__'
 
 
 class TimeEntryTypeList(DjangoObjectType):
+    duration = graphene.String()
+
     class Meta:
         model = TimeEntry
         filterset_class = TimeEntryFilter
 
 
+class SummaryDay(graphene.ObjectType):
+    date = graphene.Date()
+    duration_day = graphene.String()
+    task_list = graphene.List(TimeEntryType)
+
+    def resolve_task_list(root, info, **kwargs):
+        date = root.get('date', None)
+        user = info.context.user
+        queryset = TimeEntry.objects.filter(
+                user=user,
+                date=date
+        )
+        return queryset
+
+
+class SummaryType(graphene.ObjectType):
+    total_hours = graphene.String()
+    total_hours_day = graphene.List(SummaryDay)
+
+
 class Query(object):
     taskgroup = graphene.Field(TaskGroupType)
     taskgroup_list = DjangoFilterListField(TaskGroupListType)
     task = DjangoObjectField(TaskType)
+    task_user = graphene.List(TaskType)
     task_list = DjangoFilterListField(TaskListType)
     timeentry = DjangoObjectField(TimeEntryType)
-    timeentry_list = DjangoFilterListField(TimeEntryTypeList)
-    
\ No newline at end of file
+    summary = graphene.Field(SummaryType)
```

Comment:
Lets rename this to `weekly_summary` or something precise. And add `month_summary` on top, it looks necessary.